components/Card.vue

```diff
@@ -52,6 +52,13 @@ export default Vue.extend({
       default: () => {
         return false;
       }
+    },
+    options: {
```

Comment:
I think it would be better to avoid arrays of options for components. It makes them harder to use, imo.

There is no limit to the number of props we can have, so it would be better to split them.

I'm also not quite fond of what is essentially a switch between multiple modes being set by an arbitrary string. It makes it way harder than it needs to be to figure out what modes there are.

That "episode list mode" switch should likely be a boolean with a default of "false", then we can use it simply by going `<card :item="item" episode />`

Reply:
I've changed it as above.

When components such as people cards are added do you think it should be added with props like `cardSubtitle="prodyear/characterName/EpisodeNumber"` or left as above with multiple props?

Reply:
I think props are always the way to go :) That's what Vuetify and other Vue library do, in general. It's longer, but it's much easier to use and discover (Props will get auto-completion while string mostly values won't, maybe unless you're using an interface to define possible values, but then why wouldn't you use props instead)

Reply:
Ok, that sounds good and makes sense